SQL/Release_patches/19.1_To_20.0_upgrade.sql

```diff
@@ -161,5 +163,6 @@ UPDATE ConfigSettings
 INSERT INTO ConfigSettings (Name, Description, Visible, AllowMultiple, DataType, Parent, Label, OrderNumber) SELECT 'ComputeDeepQC', 'Determines whether a call is made from LORIS-MRI to the DeepQC app for automatic QC prediction', 1, 0, 'boolean', ID, 'Compute automatic QC', 18 FROM ConfigSettings WHERE Name="imaging_pipeline";
 
 -- default imaging_pipeline settings
-INSERT INTO Config (ConfigID, Value) SELECT ID, 0 FROM ConfigSettings cs WHERE cs.Name="ComputeDeepQC";INSERT INTO LorisMenuPermissions (MenuID, PermID)
+INSERT INTO Config (ConfigID, Value) SELECT ID, 0 FROM ConfigSettings cs WHERE cs.Name="ComputeDeepQC";
```

Comment:
@ZainVirani This is why some coding standards include policies like "file must end with a newline"

Reply:
next time I do a concat I'll add it manually, this is very difficult to review